Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 560, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
### Balance vacunas ###

if int(tasa_activos)<150:
    cumplimiento_tasa = '‚úîÔ∏è'
else:
    cumplimiento_tasa = '‚ùå'
    
if int(procesovacunacion_hoy)>= 80:
    cumplimiento_vacunacion = '‚úîÔ∏è'
else:
    cumplimiento_vacunacion = '‚ùå' 

### Descripci√≥n
desc2 = \
"""¬øToque de queda a las 00:00 hrs.? üïë

Para que en la Regi√≥n se pueda aplazar el toque de queda se deben cumplir los siguientes indicadores:

‚Ä¢ Tasa de incidencia de activos menor a 150 (actualmente es {}). {}
‚Ä¢ El porcentaje de vacunaci√≥n (cuadros completos) en poblaci√≥n objetivo debe ser mayor o igual al 80% (actualmente es {}%). {}

[ INFORMACI√ìN ADICIONAL Y FE DE ERRATAS üåå ]
‚Ä¢ Datos actualizados de acuerdo a informes del {}.
‚Ä¢ Reporte generado de forma autom√°tica. Si encuentras alg√∫n error o sugerencia, ¬°comenta!
‚Ä¢ Autor: Alejandro Dinamarca.

¬°A cuidarse TarapacaÃÅ! üòâ
""".format(tasa_activos, cumplimiento_tasa, procesovacunacion_hoy, cumplimiento_vacunacion, fecha_hoy)

## Imprimimos
print(desc2)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/tmp/ipykernel_3423/3140083791.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;31m### Balance vacunas ###[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m      2[0m [0;34m[0m[0m
[0;32m----> 3[0;31m [0;32mif[0m [0mint[0m[0;34m([0m[0mtasa_activos[0m[0;34m)[0m[0;34m<[0m[0;36m150[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      4[0m     [0mcumplimiento_tasa[0m [0;34m=[0m [0;34m'‚úîÔ∏è'[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: invalid literal for int() with base 10: '1,123'
ValueError: invalid literal for int() with base 10: '1,123'

